Update 02_connect_cloud_account.mdx (#2255)
EnterpriseDB/docs: product_docs/docs/biganimal/release/getting_started/02_connect_cloud_account.mdx

### Create Azure Active Directory Application using `create-spn`

> To simplify the process of creating an Azure AD Application, EDB provides the [`create-spn`](https://github.com/EnterpriseDB/cloud-utilities/blob/main/azure/create-spn.sh) script for Azure API users. The script automates the creation of the Active Directory Application. The script is [here](https://github.com/EnterpriseDB/cloud-utilities/tree/main/azure).

Reviewer: Do we need both links?

> To simplify the process of creating an Azure AD Application, EDB provides the [`create-spn`](https://github.com/EnterpriseDB/cloud-utilities/blob/main/azure/create-spn.sh) script for Azure API users. The script automates the creation of the Active Directory Application. The script is [here](https://github.com/EnterpriseDB/cloud-utilities/tree/main/azure).

Reviewer: ```suggestion
To simplify the process of creating an Azure AD Application, EDB provides the `create-spn` script for Azure API users (available [here](https://github.com/EnterpriseDB/cloud-utilities/blob/main/azure/create-spn.sh)). The script automates the creation of the Active Directory Application. 
```

Before using the script, ensure that these utilities are installed on your machine:

-   [jq command-line JSON processor](https://stedolan.github.io/jq/)
-   [azure cli](https://docs.microsoft.com/en-us/cli/azure/) v2.26 or above

> The alertnative way is running the script on [Azure Cloud Shell](https://azure.microsoft.com/en-us/features/cloud-shell/?ef_id=CjwKCAiA3L6PBhBvEiwAINlJ9Pj1wF0QH8KpGtC5Z4qIy21h5fcsP7tDYtBPxNLP-UMFaIiQY-EEuhoCppEQAvD_BwE%3AG%3As&OCID=AID2200202_SEM_CjwKCAiA3L6PBhBvEiwAINlJ9Pj1wF0QH8KpGtC5Z4qIy21h5fcsP7tDYtBPxNLP-UMFaIiQY-EEuhoCppEQAvD_BwE%3AG%3As&gclid=CjwKCAiA3L6PBhBvEiwAINlJ9Pj1wF0QH8KpGtC5Z4qIy21h5fcsP7tDYtBPxNLP-UMFaIiQY-EEuhoCppEQAvD_BwE#overview).

Reviewer: ```suggestion
Alternatively, you can run the script in [Azure Cloud Shell](https://azure.microsoft.com/en-us/features/cloud-shell/?ef_id=CjwKCAiA3L6PBhBvEiwAINlJ9Pj1wF0QH8KpGtC5Z4qIy21h5fcsP7tDYtBPxNLP-UMFaIiQY-EEuhoCppEQAvD_BwE%3AG%3As&OCID=AID2200202_SEM_CjwKCAiA3L6PBhBvEiwAINlJ9Pj1wF0QH8KpGtC5Z4qIy21h5fcsP7tDYtBPxNLP-UMFaIiQY-EEuhoCppEQAvD_BwE%3AG%3As&gclid=CjwKCAiA3L6PBhBvEiwAINlJ9Pj1wF0QH8KpGtC5Z4qIy21h5fcsP7tDYtBPxNLP-UMFaIiQY-EEuhoCppEQAvD_BwE#overview).
```
